randovania/games/samus_returns/logic_database/Area 8.txt

```diff
@@ -541,7 +556,10 @@ Extra - asset_id: collision_camera_010
   > Above Screw Attack Blocks
       All of the following:
           Screw Attack
-          High Jump Boots or Space Jump or Super Jump (Intermediate) or Wall Jump (Beginner)
+          Any of the following:
+              High Jump Boots or Space Jump or Single-wall Wall Jump (Beginner) or Super Jump (Intermediate) or Wall Jump (Beginner)
+              # UE
+              Morph Ball and Unmorph Extend (Beginner)
```

Comment:
I wasn't able to get this to work. This is breaking the block. Maybe it's possible, but I'd just put a todo